src/lib/main.js

```diff
@@ -89,10 +95,10 @@ import * as stateMappings from './stateMappings/index.js';
  * @property {function} [searchFunction] A hook that will be called each time a search needs to be done, with the
  * helper as a parameter. It's your responsibility to call helper.search(). This option allows you to avoid doing
  * searches at page load for example.
- * @property  {function} [createAlgoliaClient] Allows you to provide your own algolia client instead of
- * the one instantiated internally by instantsearch.js. Useful in situations where you need
- * to setup complex mechanism on the client or if you need to share it easily.
- * Usage:
+ * @property  {function} [createAlgoliaClient] _Deprecated in favor of [`searchClient`](instantsearch.html#struct-InstantSearchOptions-searchClient)._<br>
+ * Allows you to provide your own algolia client instead of the one instantiated internally by instantsearch.js.
+ * Useful in situations where you need to setup complex mechanism on the client or if you need to share it easily.
+ * <br>Usage:
```

Comment:
can you also leave a newline instead of br?

Reply:
Yes, didn't realize we could.